alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessor.java

```diff
@@ -210,7 +211,10 @@ public StaticJobAccessor(
                    .filter(distributionJobEntity -> !(distributionJobEntity.getBlackDuckJobDetails().getFilterByProject() &&
                                                           distributionJobEntity.getBlackDuckJobDetails().getProjectNamePattern() != null &&
                                                           !Pattern.matches(distributionJobEntity.getBlackDuckJobDetails().getProjectNamePattern(), projectName) &&
-                                                          !distributionJobEntity.getBlackDuckJobDetails().getBlackDuckJobProjects().contains(projectName))
+                                                          !distributionJobEntity.getBlackDuckJobDetails().getBlackDuckJobProjects()
+                                                               .stream()
+                                                               .map(BlackDuckJobProjectEntity::getProjectName)
+                                                               .anyMatch(projectName::equals))
```

Comment:
What are you checking for here? it seems you want the opposite of the `anyMatch()` result. So why not use the `noneMatch()` method?